shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -521,43 +515,37 @@ def _set_quantity__check_product_in_line(
     def _set_quantity__check_quantity_done(
         self, move_line, location=None, package=None, confirmation=None
     ):
-        rounding = move_line.product_id.uom_id.rounding
-        qty_done = move_line.qty_done
-        qty_todo = move_line.product_uom_qty
-        # If qty done is >= qty todo, then there's nothing more to pick
-        if float_compare(qty_done, qty_todo, precision_rounding=rounding) > 0:
-            message = self.msg_store.unable_to_pick_more(qty_todo)
+        stock = self._actions_for("stock")
+        if not stock.move_line_check_qty_picked(move_line):
```

Comment:
This method is added in the following commit (i), but it doesn't exist up until this one (i-1).
My recommendation then: merge the two commits